acquisitions/src/main/resources/thunderjet/mod-orders/features/pieces-batch-update-status.feature

```diff
@@ -0,0 +1,98 @@
+Feature: Update Pieces statuses in batch
+
+  Background:
+    * url baseUrl
+
+    * callonce loginAdmin testAdmin
+    * def okapitokenAdmin = okapitoken
+    * callonce loginRegularUser testUser
+    * def okapitokenUser = okapitoken
+    * def headersAdmin = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenAdmin)', 'Accept': 'application/json'  }
+    * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': '*/*'  }
+    * configure retry = { count: 5, interval: 1000 }
+    * configure headers = headersAdmin
+
+    * callonce variables
+    * def fundId = callonce uuid
+    * def budgetId = callonce uuid1
+    * def orderId = callonce uuid2
+    * def poLineId = callonce uuid3
+    * def titleId = callonce uuid4
+    * def pieceId1 = callonce uuid5
+    * def pieceId2 = callonce uuid6
+    * def pieceId3 = callonce uuid7
+
+    ### Before All ###
+    # 1. Prepare finance data
+    * def v = callonce createFund { id: '#(fundId)' }
+    * def v = callonce createBudget { id: '#(budgetId)', allocated: 100, fundId: '#(fundId)', status: 'Active' }
+
+    # 2. Prepare acquisitions data
+    * def v = callonce createOrder { id: '#(orderId)' }
+    * def v = callonce createOrderLine { id: '#(poLineId)', orderId: '#(orderId)', fundId: '#(fundId)', isPackage: true }
+    * def v = callonce openOrder { orderId: '#(orderId)' }
+    * def v = callonce createTitle { titleId: '#(titleId)', poLineId: '#(poLineId)' }
+
+    # 3. Prepare pieces data
+    * table pieceData
+      | pieceId  | titleId | poLineId |
+      | pieceId1 | titleId | poLineId |
+      | pieceId2 | titleId | poLineId |
+      | pieceId3 | titleId | poLineId |
+    * def v = callonce createPiece pieceData
+    * configure headers = headersUser
+
+  @Positive
+  Scenario: Update once Piece status in batch to received and verify PoLine receipt status
+    # 1. Update once Piece status in batch to "Unreceivable"
+    * def v = call updatePiecesBatchStatus { pieceIds: ["#(pieceId1)"], receivingStatus: 'Unreceivable' }
+    # 2. Verify PoLine receipt status is "Partially Received"
+    * def v = call read('@verifyPoLineReceiptStatus') { _poLineId: '#(poLineId)', _receiptStatus: 'Partially Received' }
```

Comment:
would be good to add assertions that pieces also has correct status.
Also if time permits we can call mod-audit, it has endpoint about piece status changes